manifests/profile/haproxy/keepalived.pp

```diff
@@ -22,7 +22,8 @@
     require    => Package['keepalived'],
   }
 
-  $frontends = balanced_frontends()
+  $nodes_for_class = nodes_for_class($title)
```

Comment:
I'm not sure this will work, but I'm definitely interested. I'd thought that classes don't have the `$title` parameter, but I never actually checked. If it don't work in production, it still passes the tests because our `nodes_for_class` mock doesn't pay attention to its input.

Reply:
It seems to work as expected in production